rotkehlchen/tests/unit/decoders/test_aave.py

```diff
@@ -216,3 +223,668 @@ def test_aave_eth_withdraw_v1(database, ethereum_inquirer):
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = string_to_evm_address('0x2715273613632226985186221669179813245119')
+    transaction = EvmTransaction(
+        tx_hash=evmhash,
+        chain_id=ChainID.ETHEREUM,
+        timestamp=0,
+        block_number=0,
+        from_address=user_address,
+        to_address=ethereum_inquirer.contracts.contract('AAVE_V2_LENDING_POOL').address,
+        value=0,
+        gas=0,
+        gas_price=0,
+        gas_used=0,
+        input_data=b'',
+        nonce=0,
+    )
+    receipt = EvmTxReceipt(
+        tx_hash=evmhash,
+        chain_id=ChainID.ETHEREUM,
+        contract_address=None,
+        status=True,
+        type=0,
+        logs=[
+            EvmTxReceiptLog(
+                log_index=251,
+                data=hexstring_to_bytes('0x'),
+                address=string_to_evm_address('0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0x00058a56ea94653cdf4f152d227ace22d4c00ad99e2a43f58cb7d9e3feb295f2'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000002715273613632226985186221669179813245119'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbevmtx = DBEvmTx(database)
+    with dbevmtx.db.user_write() as cursor:
+        dbevmtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
+    decoder = EthereumTransactionDecoder(
+        database=database,
+        ethereum_inquirer=ethereum_inquirer,
+        transactions=eth_transactions,
+    )
+    with dbevmtx.db.user_write() as cursor:
+        events = decoder.decode_transaction(cursor, transaction=transaction, tx_receipt=receipt)
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO, usd_value=ZERO),
+            location_label=user_address,
+            notes='Burned 0 ETH for gas',
+            counterparty=CPT_GAS,
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=251,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_WETH,
+            balance=Balance(),
+            location_label=None,
+            notes='Enable WETH (WETH) as collateral on AAVE v2 for user 0x2715273613632226985186221669179813245119',  # noqa: E501
```

Comment:
You can use the variable for the user address
```suggestion
            notes=f'Enable WETH (WETH) as collateral on AAVE v2 for user {user_address}',  # noqa: E501
```